includes/utils.inc

```diff
@@ -1242,6 +1229,37 @@ function wf_crm_get_fields($var = 'fields') {
           'empty_option' => t('None'),
         );
       }
+      // Membership Date Rule Fields
+      $fields['membership_start_date_rules'] = array(
+        'name' => t('Start Date'),
+        'type' => 'select',
+        'expose_list' => TRUE,
+        'value' => 0,
+        'exposed_empty_option' => t('Automatic'),
+      );
+      $fields['membership_start_date_memberships'] = array (
```

Comment:
maybe membership_start_date_rules_memberships is more clear, same for end_date_rules